### PR TITLE
Add dugtrio for servicemonitor

### PR DESCRIPTION
**Pull request by aBMania**
Hello, dugtrio is pushing metrics but no service monitor was setup.

I've tried it, and it works correctly

### CODE REVIEW

File: charts/dugtrio/values.yaml [hunk 1]
```diff
@@ -141,6 +141,28 @@ extraPorts: []
 # -- Additional env variables
 extraEnv: []
 
+serviceMonitor:
+  # -- If true, a ServiceMonitor CRD is created for a prometheus operator
+  # https://github.com/coreos/prometheus-operator
+  enabled: true
```
Comment (samcm) [inline charts/dugtrio/values.yaml:147]: Could we flip this to `false` by default to align with the other charts please? This way users who don't have the prometheus CRDs installed are fine with the defaults 🙏

**Comment by samcm** [comment]
Thanks @aBMania! Just added 1 comment. Will need you to bump the chart version in `Chart.yaml` as well and then we can merge 🎉

**Comment by aBMania** [comment]
Updated ! :)

**Comment by pk910** [comment]
please run `make docs` to update the README :)

**Comment by barnabasbusa** [comment]
Closing in favor of #299

**Pull request closed without merging**